src/lib/eventBus.js

```diff
@@ -0,0 +1,3 @@
+import Vue from 'vue';
+
+export const eventBus = new Vue();
```

Comment:
Not sure I understand this, could you elaborate on what you are doing?

Reply:
Here's an old-school approach of pub-sub in Vue. Now I should use this.$root.$emit() and this.$root.$on() instead